README.md

```diff
@@ -66,6 +66,8 @@
 
 > **Everything here is created on rofi version : `1.7.4`**
 
+### From source
+
```

Comment:
Here you added a space between the title and description, but below you don't have a space between the title and description, could you decide on one so that there is consistency in the documentation?

![image](https://github.com/adi1090x/rofi/assets/66505715/10abfd54-8d21-45a6-8ec7-65397b660c70)

Reply:
Thanks for the review, fixed in 1eefe3bbdfd2f69418c24d52b4361cfe5871f9de